### PR TITLE
[WIP] IWSLT2016 Recipe

### PR DESCRIPTION
**Pull request by butsugiri**
This PR adds recipe for the IWSLT2016 Machine Translation.

- [Task Detail](https://sites.google.com/site/iwsltevaluation2016/mt-track)
- [Dataset](https://wit3.fbk.eu/mt.php?release=2016-01)

Currently this PR is just a copy of `must_c mt1` recipe.
I will modify the code (maybe after completing my experiments).

### CODE REVIEW

**Comment by butsugiri** [comment]
Rest of TODOs:

- [ ] Add English as target language (currently En-->De only)
- [ ] Take BPE merge operations as variable (currently fixed to 16k)

**Comment by butsugiri** [comment]
^I completed rest of TODOs.
@sw005320 @hirofumi0810 Can you tell me what to do next?

File: egs/iwslt16/mt1/local/generate_json.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+ESPNetのMT実装が要求するフォーマットでtrain/devデータを作る．
```
Comment (sw005320) [inline egs/iwslt16/mt1/local/generate_json.py]: English comments are better...

Comment (butsugiri) [reply egs/iwslt16/mt1/local/generate_json.py]: Fixed 🙏

**Comment by sw005320** [comment]
there some errors in the ci shell check https://travis-ci.org/espnet/espnet/jobs/643679572#L2379
please fix them.

**Comment by butsugiri** [comment]
I fixed the shellcheck error.

**Comment by sw005320** [comment]
This is a great job!
Thanks!

**Pull request merged**